sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md

```diff
@@ -8,112 +8,313 @@ The authentication events trigger for Azure Functions allows you to implement a
 
-Install the authentication events trigger for Azure Functions with [NuGet](https://www.nuget.org/):
+### Prerequisites
 
-```dotnetcli
-dotnet add package Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents
-```
+- A basic understanding of the concepts covered in [Custom authentication extensions overview](https://learn.microsoft.com/entra/identity-platform/custom-extension-overview).
+- An Azure subscription with the ability to create Azure Functions. If you don't have an existing Azure account, sign up for a [free trial](https://azure.microsoft.com/free/dotnet/) or use your [Visual Studio Subscription](https://visualstudio.microsoft.com/subscriptions/) benefits when you [create an account](https://account.windowsazure.com/Home/Index).
+- A Microsoft Entra ID tenant. You can use either a customer or workforce tenant for this how-to guide.
+- One of the following IDEs and configurations:
+    - Visual Studio with [Azure Development workload for Visual Studio](https://learn.microsoft.com/dotnet/azure/configure-visual-studio) configured.
+    - Visual Studio Code, with the [Azure Functions](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions) extension enabled.
 
-### Prerequisites
+### Create and build the Azure Function app
 
-- **Azure Subscription:** To use Azure services, including Azure Functions, you'll need a subscription. If you do not have an existing Azure account, you may sign up for a [free trial](https://azure.microsoft.com/free/dotnet/) or use your [Visual Studio Subscription](https://visualstudio.microsoft.com/subscriptions/) benefits when you [create an account](https://account.windowsazure.com/Home/Index).
+In this step, you create an HTTP trigger function API using your IDE, install the required NuGet packages and copy in the sample code. You build the project and run the function to extract the local function URL.
 
-### Authenticate the client
+### Create the application
 
-When the Microsoft Entra authentication events service calls your custom extension, it sends an `Authorization` header with a `Bearer {token}`. This token represents a [service to service authentication](https://learn.microsoft.com/azure/active-directory/develop/v2-oauth2-client-creds-grant-flow) in which:
+To create an Azure Function app, follow these steps:
 
-* The '**resource**', also known as the **audience**, is the application that you register to represent your API. This is represented by the `aud` claim in the token.
-* The '**client**' is a Microsoft application that represents the Microsoft Entra authentication events service. It has an `appId` value of `99045fe1-7639-4a75-9d4a-577b6ca3810f`. This is represented by:
-  * The `azp` claim in the token if your application `accessTokenAcceptedVersion` property is set to `2`.
-  * The `appid` claim in the token if your resource application's `accessTokenAcceptedVersion` property is set to `1` or `null`.
+#### [Visual Studio](#tab/visual-studio)
```

Comment:
No objection to including them, but this is a deeper level than we typically go.   Generally the assumption would be that a developer knows how to work with Azure Functions, but is brand new to your specific extension package.

Reply:
Alright, I have removed the step-by-step instructions for creating a function in IDE and installing the package